From 51ec4fd847369bc77e72850fe93d727ae9c656b9 Mon Sep 17 00:00:00 2001
From: Szymon Janc <szymon.janc@tieto.com>
Date: Fri, 19 Sep 2014 16:10:07 +0200
Subject: [PATCH] tools: Fix compilation errors with musl

This fix number of includes for <sys/poll.h> instead of <poll.h>

  CC       tools/hciattach.o
In file included from tools/hciattach.c:41:0:
/usr/include/sys/poll.h:1:2: error: #warning redirecting incorrect
   #include <sys/poll.h> to <poll.h> [-Werror=cpp]
 #warning redirecting incorrect #include <sys/poll.h> to <poll.h>
  ^
---
 tools/amptest.c            | 2 +-
 tools/btattach.c           | 2 +-
 tools/btsnoop.c            | 1 +
 tools/ciptool.c            | 2 +-
 tools/cltest.c             | 2 +-
 tools/hciattach.c          | 2 +-
 tools/hciattach_qualcomm.c | 2 +-
 tools/hciattach_tialt.c    | 2 +-
 tools/hcidump.c            | 2 +-
 tools/l2ping.c             | 2 +-
 tools/l2test.c             | 2 +-
 tools/rfcomm.c             | 2 +-
 12 files changed, 12 insertions(+), 11 deletions(-)

diff --git a/tools/amptest.c b/tools/amptest.c
index 6192f7e..bba0a9a 100644
--- a/tools/amptest.c
+++ b/tools/amptest.c
@@ -32,7 +32,7 @@
 #include <alloca.h>
 #include <stdlib.h>
 #include <stdbool.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 
diff --git a/tools/btattach.c b/tools/btattach.c
index 9458fd7..bdbbe16 100644
--- a/tools/btattach.c
+++ b/tools/btattach.c
@@ -35,7 +35,7 @@
 #include <getopt.h>
 #include <termios.h>
 #include <sys/ioctl.h>
-#include <sys/poll.h>
+#include <poll.h>
 
 #include <bluetooth/bluetooth.h>
 #include <bluetooth/hci.h>
diff --git a/tools/btsnoop.c b/tools/btsnoop.c
index 14581f2..71191eb 100644
--- a/tools/btsnoop.c
+++ b/tools/btsnoop.c
@@ -35,6 +35,7 @@
 #include <stdbool.h>
 #include <string.h>
 #include <getopt.h>
+#include <endian.h>
 #include <arpa/inet.h>
 #include <sys/stat.h>
 
diff --git a/tools/ciptool.c b/tools/ciptool.c
index 72338dc..3ff9fb1 100644
--- a/tools/ciptool.c
+++ b/tools/ciptool.c
@@ -32,7 +32,7 @@
 #include <string.h>
 #include <getopt.h>
 #include <signal.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 
diff --git a/tools/cltest.c b/tools/cltest.c
index 4ddb98a..0231805 100644
--- a/tools/cltest.c
+++ b/tools/cltest.c
@@ -32,7 +32,7 @@
 #include <alloca.h>
 #include <stdlib.h>
 #include <stdbool.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 
diff --git a/tools/hciattach.c b/tools/hciattach.c
index 1904ac5..d8ef7e7 100644
--- a/tools/hciattach.c
+++ b/tools/hciattach.c
@@ -37,8 +37,8 @@
 #include <syslog.h>
 #include <termios.h>
 #include <time.h>
+#include <poll.h>
 #include <sys/time.h>
-#include <sys/poll.h>
 #include <sys/param.h>
 #include <sys/ioctl.h>
 
diff --git a/tools/hciattach_qualcomm.c b/tools/hciattach_qualcomm.c
index eb72a0f..0e25905 100644
--- a/tools/hciattach_qualcomm.c
+++ b/tools/hciattach_qualcomm.c
@@ -36,8 +36,8 @@
 #include <syslog.h>
 #include <termios.h>
 #include <time.h>
+#include <poll.h>
 #include <sys/time.h>
-#include <sys/poll.h>
 #include <sys/param.h>
 #include <sys/ioctl.h>
 #include <sys/uio.h>
diff --git a/tools/hciattach_tialt.c b/tools/hciattach_tialt.c
index 5c7f3a5..a7c1706 100644
--- a/tools/hciattach_tialt.c
+++ b/tools/hciattach_tialt.c
@@ -35,8 +35,8 @@
 #include <syslog.h>
 #include <termios.h>
 #include <time.h>
+#include <poll.h>
 #include <sys/time.h>
-#include <sys/poll.h>
 #include <sys/param.h>
 #include <sys/ioctl.h>
 #include <sys/uio.h>
diff --git a/tools/hcidump.c b/tools/hcidump.c
index 37a9f00..8839eb0 100644
--- a/tools/hcidump.c
+++ b/tools/hcidump.c
@@ -33,7 +33,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <getopt.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <sys/ioctl.h>
diff --git a/tools/l2ping.c b/tools/l2ping.c
index 29fb3d0..3dd437e 100644
--- a/tools/l2ping.c
+++ b/tools/l2ping.c
@@ -35,7 +35,7 @@
 #include <getopt.h>
 #include <signal.h>
 #include <sys/time.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/socket.h>
 
 #include <bluetooth/bluetooth.h>
diff --git a/tools/l2test.c b/tools/l2test.c
index 67ca70a..71ad109 100644
--- a/tools/l2test.c
+++ b/tools/l2test.c
@@ -38,7 +38,7 @@
 #include <syslog.h>
 #include <signal.h>
 #include <sys/time.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 
diff --git a/tools/rfcomm.c b/tools/rfcomm.c
index b5bea38..659bbec 100644
--- a/tools/rfcomm.c
+++ b/tools/rfcomm.c
@@ -34,7 +34,7 @@
 #include <getopt.h>
 #include <signal.h>
 #include <termios.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/param.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
-- 
2.0.4

